python/Ganga/Core/GangaRepository/Registry.py

```diff
@@ -189,9 +218,18 @@ def stop(self):
 
     @property
     def stopped(self):
+        """
+        Returns if the flusher has stopped as a boolean?
+        """
         return self._stop.isSet()
 
     def join(self, *args, **kwargs):
+        """
+        Not clear if this is called anywhere
```

Comment:
This is an unhelpful code comment. `join` is implemented since we are subclassing `Thread` and so must make sure that if `join` is called on it, that it works in a polymorphic way and does the correct thing.